### ISSUE TITLE
TestExitTime intermittent failure

### ISSUE THREAD

**Issue by nguerrera** [dotnet/corefx#2660]
```
     System.Diagnostics.ProcessTests.ProcessTests.TestExitTime [FAIL]
        Assert+WrapperXunitException : File path: d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTests.cs. Line: 140
        ---- TestExitTime is incorrect.
        Expected: True
        Actual:   False
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(582,0): at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(577,0): at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTests.cs(140,0): at System.Diagnostics.ProcessTests.ProcessTests.TestExitTime()
           ----- Inner Stack Trace -----
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(574,0): at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
```

``` C#
[Fact] 
public void TestExitTime() 
{ 
    DateTime timeBeforeProcessStart = DateTime.UtcNow; 
    Process p = CreateProcessInfinite(); 
    p.Start(); 
    Assert.Throws<InvalidOperationException>(() => p.ExitTime); 
    p.Kill(); 
    Assert.True(p.WaitForExit(WaitInMS)); 
    Assert.True(p.ExitTime.ToUniversalTime() > timeBeforeProcessStart, "TestExitTime is incorrect."); 
}
```

That should be >= not >, but am I really that (un)lucky today or is there something else here?

**Comment by nguerrera** [comment]
#2655 hit this

**Comment by nguerrera** [comment]
Presumed fixed by #2665
